.github/workflows/formatting-changes.yml

```diff
@@ -0,0 +1,49 @@
+name: Formatting changes
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  format:
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.pull_request.head.ref }}
+          token: ${{ secrets.GIX_CREATE_PR_PAT }}
+
+      - name: Restore npm cache
```

Comment:
this is not needed anymore, right?

Reply:
i mean, shall it be removed from here, right?

Reply:
sorry, forgot to push... should be updated now